### PR TITLE
required to display  the `coinid` for the `vout` 

### LINKED ISSUE #284: "coinid" missing in the vout type of "MEER Asset"
```bash
{
  "vout": [
    {
      "coin": "MEER Balance",
      "coinid": 1,
      "amount": 6600000,
      "scriptPubKey": {
        "addresses": [
          "Mk6qtzkdEfykNzDsXavCGhNwgeaToTNV1ZDfJgLRjdPdLq3nxPA7R"
        ]
      },
      "to": "0x0b334490F0C148Fad70184ed87f971f0186909D9"
    },
    {
      "coin": "MEER Asset",
      "amount": 9394000,
      "scriptPubKey": {
        "asm": "OP_DUP OP_HASH160 6cb6dd6f712fbbde938eeaded72fc51c5b5116e5 OP_EQUALVERIFY OP_CHECKSIG",
        "hex": "76a9146cb6dd6f712fbbde938eeaded72fc51c5b5116e588ac",
        "reqSigs": 1,
        "type": "pubkeyhash",
        "addresses": [
          "MmVWA3QHkAJnAf2ZzG3CSXsf4kQBHawTJpf"
        ]
      }
    }
  ],
  "blockhash": "4e1db8f60c7123aa0b9eece413dc8639110f56156dd5078020cf0701e195d4f2",
  "confirmations": 50862,
  "txsvalid": true,
  "type": "TxTypeCrossChainExport"
}
```

### PR DESCRIPTION
**Pull request by jamesvan2019**
Fix #284

### CODE REVIEW

File: core/json/tx.go [hunk 1]
```diff
@@ -102,7 +102,7 @@ func (v *Vin) MarshalJSON() ([]byte, error) {
 // getrawtransaction and decoderawtransaction use the same structure.
 type Vout struct {
 	Coin         string             `json:"coin,omitempty"`
-	CoinId       uint16             `json:"coinid,omitempty"`
```
Comment (lochjin) [inline core/json/tx.go:105]: why not only remove the `omitempty` of tag ?

Comment (lochjin) [inline core/json/tx.go:105]: why not only remove the omitempty of tag ?

**Review by lochjin — CHANGES_REQUESTED** [review]

File: core/json/tx.go (hunk 1 above)
Comment (jamesvan2019) [reply core/json/tx.go:105]: In order not to affect the previous lock vout

```json
{
            "scriptPubKey": {
              "asm": "OP_RETURN OP_MEER_LOCK 007841cb02",
              "hex": "6ac305007841cb02",
              "type": "LockAmount"
            }
}
```
if remove omitempty, the lock vout will be this

```json
{
            "coinid": 0,
            "scriptPubKey": {
              "asm": "OP_RETURN OP_MEER_LOCK 007841cb02",
              "hex": "6ac305007841cb02",
              "type": "LockAmount"
            }
}
```

**Review by dindinw — APPROVED** [review]

File: core/json/tx.go (hunk 1 above)
Comment (dindinw) [reply core/json/tx.go:105]: removing the `omitempty` is enough for us

**Review by lochjin — APPROVED** [review]

**Pull request merged**